pkg/cmd/openshift-controller-manager/configdefault/openshiftcontrolplane_default.go

```diff
@@ -46,4 +46,12 @@ func SetRecommendedOpenShiftControllerConfigDefaults(config *openshiftcontrolpla
 	if config.ImageImport.ScheduledImageImportMinimumIntervalSeconds == 0 {
 		config.ImageImport.ScheduledImageImportMinimumIntervalSeconds = 15 * 60 // 15 minutes
 	}
+
+	configdefaults.DefaultStringSlice(&config.ServiceAccount.ManagedNames, []string{"builder", "deployer"})
+
+	// TODO this default is WRONG, but it appears to work
+	configdefaults.DefaultString(&config.Deployer.ImageTemplateFormat.Format, "openshift/origin-${component}:${version}")
+
+	// TODO this default is WRONG, but it appears to work
+	configdefaults.DefaultString(&config.Build.ImageTemplateFormat.Format, "openshift/origin-${component}:${version}")
```

Comment:
i'd prefer to see this be fully qualified (quay.io or docker.io)?

Reply:
> i'd prefer to see this be fully qualified (quay.io or docker.io)?

done